### ISSUE TITLE
[Bug] Placing lockboxes

### ISSUE THREAD

**Issue by crazymike10** [EpochModTeam/Epoch#141]
0.3 build - Group creator places down lock box, group member can unlock but cannot lock. Group member places down lockbox, group creator and member can unlock and lock.

**Comment by crazymike10** [comment]
As well, a person cannot pack the lockbox.

**Comment by crazymike10** [comment]
Found out lockboxes (once completely full) will not hold items. Had two lockboxes full of random items, now left with quarter full. 

Let me know if you need more info on this, or if you need further testing.

**Comment by vbawol** [comment]
Packing of lockboxes is fixed. 
Does this happen only after a server restart?

**Comment by vbawol** [comment]
if the issue with lost items exists with 0.3.0.1 please open another issue.
